burstphoto/align.swift

```diff
@@ -106,7 +106,7 @@ func perform_denoising(image_urls: [URL], progress: ProcessingProgress, ref_idx:
     
     // ensure that all files are .dng, converting them if necessary
     var dng_urls = image_urls
-    let convert_to_dng = image_extension != "dng"
+    let convert_to_dng = image_extension.lowercased() != "dng"
```

Comment:
I mean that you should do a literal merge of main into this branch, and not the changes within it. Github is going to keep complaining about merge conflicts that will still have to be dealt with